src/components/navbar.tsx

```diff
@@ -0,0 +1,53 @@
+import SearchIcon from "./svgs/search";
+import ProfileIcon from "./svgs/profile";
+import AppointmentsIcon from "./svgs/appointments";
+import ReceiptsIcon from "./svgs/receipts";
+import React from "react";
+
+type IconButtonProps = {
+    icon: React.ReactNode; 
+    label: string;
+};
+
+function IconButton({ icon, label }: IconButtonProps) {
+    return (
+        <div className="group text-white flex flex-col items-center p-4 rounded-mini hover:bg-teal-light">
+            <div className="group-hover:text-white-dark">
+                {icon}
+            </div>
+            <h1 className="text-sm mt-2 group-hover:text-white-dark">{label}</h1>
+        </div>
+    );
+}
+
+
+export default function NavBar() {
+    return (
+        <div className="bg-teal fixed bottom-0 w-full flex flex-row pb-2 pt-2 justify-around items-center  rounded-t-custom">
```

Comment:
I believe the figma has the colors of the Appointments button in the navbar inverted because that's supposed to represent the selected page, so add some state to keep track of the current page and appropriately invert the color of the icons to indicate the current page.

If you are unsure how to do this, we can get on a call and work it out.